django_project_base/settings.py

```diff
@@ -0,0 +1,11 @@
+SETTINGS = (
```

Comment:
Tukaj ti manjka MODULE_SETTINGS_PREFIX = "PROJECT_BASE_" in didajanje tega prefixa vsem settingom.

Manjka dokumentacija teh settingov v .rst.

Reply:
done